presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaMetadata.java

```diff
@@ -186,8 +185,7 @@ public ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connect
                 columns.put(tableName, getTableMetadata(tableName).getColumns());
             }
             catch (TableNotFoundException e) {
-                // Normally it would mean the table disappeared during listing operation
-                throw new IllegalStateException(format("Table %s cannot be gone because tables are statically defined", tableName), e);
+                // information_schema table or the the table disappeared during listing operation
```

Comment:
`information_schema table ... disappeared during listing operation`?
Can that really happen?

Reply:
it's an "or"

Reply:
Yes, but can `information_schema table ... disappeared during listing operation` part of `OR` happen?

Reply:
the "disappear" cannot happen. I changed the wording here